tests/README.md

```diff
@@ -277,7 +280,7 @@ common code to the shared directory.
 # Running tests
 
 To test you profile or rule use `test_suite.py` script. It can take your SCAP source data stream, and test it on the specified backend.
```

Comment:
Don't we want to rather recommend using `automatus.py` instead of the `test_suite.py` as we would like to get rid of this wrapper in the future?